app/models/miq_queue/constants.rb

```diff
@@ -0,0 +1,45 @@
+module MiqQueueConstants
```

Comment:
As an FYI, I couldn't nest modules here because `MiqQueue` is a class, and if load order loads this first, and then tries to load `app/models/miq_queue.rb`, there will be an error.  Hopefully other Rails conventions because of this won't bite us with this.